.kiro/specs/workspace-index-consolidation/design.md

```
@@ -0,0 +1,668 @@
+            file_cache: &self.file_cache,
+        }
+    }
+}
+```
+
+## Data Models
+
+### DocumentState Fields
+
+| Field | Type | Purpose |
+|-------|------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)